content/250-postgres/300-database/400-connection-pooling.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: 'Connection pooling'
+metaTitle: 'Connection pooling in Prisma Postgres'
+metaDescription: 'Learn about connection pooling in Prisma Postgres'
+tocDepth: 3
+toc: true
+---
+
+Prisma Postgres provides built-in connection pooling by default, enabled by [Prisma Accelerate](/accelerate/). By using Prisma Postgres, you get the benefits of connection pooling without having to configure anything. The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable.
+
+In some cases, however, it may be beneficial to further configure connection pooling in order to optimize the performance of your application.
+
+:::note
+
+This document focuses on the connection pooling features of Prisma Postgres. For more information about the internal connection pool of Prisma ORM specifically, see our [ORM connection pooling documentation](/orm/prisma-client/setup-and-configuration/databases-connections/connection-pool).
+
+:::
+
+:::info
+
+To learn more about connection pooling generally, be sure to check out [Prisma's Data Guide](https://www.prisma.io/dataguide/database-tools/connection-pooling) for a comprehensive overview on what connection pooling is, what it seeks to address, and how it works.
+
+:::
```

Comment:
let's delete this entire info box in favor of a simple hyperlink in the intro sentence 🙏 

```suggestion
```